### PR TITLE
Added RHOCMClusterID

### PR DESCRIPTION
**Pull request by ngraham20**
Jira issue: https://issues.redhat.com/browse/ACM-9338

### CODE REVIEW

File: config/crd/bases/discovery.open-cluster-management.io_discoveredclusters.yaml [hunk 1]
```diff
@@ -160,6 +162,8 @@ spec:
                 type: string
               cloudProvider:
                 type: string
+              rhocmClusterId:
+                type: string
```
Comment (dislbenn) [inline config/crd/bases/discovery.open-cluster-management.io_discoveredclusters.yaml:166]: You are providing `rhocmClusterId` twice. I believe this one should be `ocpClusterId`.

Comment (ngraham20) [reply config/crd/bases/discovery.open-cluster-management.io_discoveredclusters.yaml:166]: oh nope, I just forgot to add it entirely :sweat_smile:

**Pull request merged**